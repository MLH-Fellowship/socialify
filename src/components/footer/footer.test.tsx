import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Footer from './footer'

test('Footer renders', () => {
  const footer = shallow(<Footer />)

  expect(toJson(footer)).toMatchSnapshot()
})
