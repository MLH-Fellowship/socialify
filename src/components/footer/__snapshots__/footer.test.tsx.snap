// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders 1`] = `
<div
  className="footerFlex"
>
  <Text
    style={
      Object {
        "margin": "unset",
      }
    }
  >
    Made with 
    <HeartSvg />
     by 
    <ForwardRef(Link)
      href="https://cryogenicplanet.tech"
      rel="noopener noreferrer"
      target="_blank"
    >
      CryogenicPlanet
    </ForwardRef(Link)>
     and 
    <ForwardRef(Link)
      href="https://github.com/wei/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Wei
    </ForwardRef(Link)>
  </Text>
</div>
`;
